app/views/transactionDetails.html

```diff
@@ -180,6 +183,11 @@ <h6 class="subtitle"><em>Received at {{transactionDetails.response.timestamp | d
                     <div class="hljsFormat" hljs source="transactionDetails.response.body"></div>
                     <button style="margin-top: 10px;" class="btn btn-primary pull-right" ng-click="viewBodyDetails('Response', transactionDetails.response.body, transactionDetails.response.headers);">View Full Response Body</button>
                   </div>
+                  <div ng-if="transactionDetails.error" style="display: inline-block; width: 100%; margin-top: 15px;">
+                    <h4>Error</h4>
+                    <div>{{transactionDetails.error.message}}</div>
```

Comment:
Maybe this should be the same as line 183? <div class="hljsFormat" hljs source="transactionDetails.error.message"></div>